bluesky_queueserver/manager.py

```diff
@@ -329,6 +402,11 @@ async def _print_db_uids_handler(self, request):
         self._print_db_uids()
         return {"success": True, "msg": ""}
 
+    async def _kill_manager_handler(self, request):
+        # This is expected to block the event loop forever
+        while True:
+            pass
```

Comment:
Should there be a sleep in this loop to avoiding pinning a CPU?

Reply:
Yes, sleep will work.